rds/sql/sql_test/PostgreSQL_Lowercase_All_Common_PK_Col_Named_ID/usp-create-person-demographics.sql.golden

```diff
@@ -52,6 +52,12 @@ AS $BODY$
 
   $BODY$;
   -- ALTER PROCEDURE usp_create_person_demographics(TEXT, TEXT, TEXT, TEXT, TIMESTAMPTZ, INTEGER, INTEGER, TEXT, TEXT, TEXT, TEXT, TEXT, TEXT, TEXT, INTEGER) OWNER TO (TODO: OWNER);
+  INSERT INTO plpgsql_check_result
+  (function_id, line_no, "statement", sql_state, message, detail, hint,
+   "level", "position", query, context)
```

Comment:
@Kunjahamed-P @divyaSkumarCI01188 please check semicolon at end of insert; without it, the code will break?

Reply:
@shah 
We have already added [semicolon at the end of the insert](https://github.com/gov-suite/governed-im-transform/blob/master/rds/sql/sql_test/PostgreSQL_Lowercase_All_Common_PK_Col_Named_ID/usp-create-person-demographics.sql.golden#L60).